python/cuml/test/test_rf_printing.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+from cuml.ensemble import RandomForestClassifier as RFClassifier
+from cuml.test.utils import get_handle
+from sklearn.datasets import make_classification
+
+
+@pytest.mark.parametrize('n_estimators', [5, 10, 20])
+@pytest.mark.parametrize('detailed_printing', [True, False])
+def test_rf_printing(capfd, n_estimators, detailed_printing):
+
+    X, y = make_classification(n_samples=500, n_features=10,
```

Comment:
in the test_random_forest.py there is a common function used to generate the datatsets. It will help keep duplicated code in the test file to a minimum.

Reply:
@Salonijain27 I don't think `small_clf` would help here, as we don't really care about the dataset size.

Reply:
I have moved the code to the `test_random_forest.py`. However, I could not use `small_clf` along such that printing test would be skipped for quality and stress parameters. So leaving it as it is.